includes/callbacks.inc

```diff
@@ -64,19 +64,19 @@ function islandora_entities_autocomplete($type) {
   $mappings = array(
     'scholar' => array(
       'cmodel' => 'islandora:personCModel',
-      'title' => 'MADS_title_mt',
+      'title' => 'islandora_entities_title_collection',
```

Comment:
These should reference the new variables such as `variable_get('islandora_entities_scholar_title_solr_field`, 'MADS_title_mt')`.